frame/staking/src/lib.rs

```diff
@@ -1237,6 +1312,42 @@ decl_module! {
 			<Self as Store>::UnappliedSlashes::insert(&era, &unapplied);
 		}
 
+		/// Make one staker's payout for one era.
+		///
+		/// - `who` is the nominator to pay out.
+		/// - `era` may not be lower than one following the most recently paid era. If it is higher,
+		///   then it indicates an instruction to skip the payout of all previous eras.
+		/// - `validators` is the list of all validators that `who` had exposure to during `era`.
+		///   If it is incomplete, then less than the full reward will be paid out.
+		///
+		/// WARNING: Incorrect arguments here can result in loss of payout. Be very careful.
+		///
+		/// 1 balance transfer
+		/// Up to 16 storage reads, each of `O(N)` size and decode complexity; `N` is maximum
+		/// nominations that can be given to a single validator. (`MAX_NOMINATIONS` is the maximum
+		/// number of validators that may be nominated by a single nominator.) This is bounded only
+		/// economically (all nominators are required to place a minimum stake).
+		/// Compute: O(MAX_NOMINATIONS * logN).
+		// TODO: Limit the amount of nominators that can be assigned to a validator by Phragmen.
+		fn payout_nominator(origin, era: EraIndex, validators: Vec<T::AccountId>)
+			-> DispatchResult
+		{
+			let who = ensure_signed(origin)?;
+			Self::do_payout_nominator(who, era, validators)
+		}
+
+		/// Make one staker's payout for one era.
+		///
+		/// - `who` is the nominator to pay out.
```

Comment:
Could you specify (for this and `payout_nominator`) that this function should be called by the _Controller?_
```suggestion
		/// - `who` is the validator to pay out.
```

Reply:
yes definitely, done